### ISSUE TITLE
[BUG][Functional] Root user cannot remove MFA device

### ISSUE THREAD

**Issue by evollebekk** [aws-samples/aws-secure-environment-accelerator#1102]
**Required Basic Info**
- Accelerator Version:  v1.5.5
- Install Type: Upgrade
- Upgrade from version: v1.5.4-a

**Describe the bug**
The root user of accounts in the Sensitive OU is not able to remove MFA devices 

**Failure Info**
- What error messages have you identified, if any:  User: arn:aws:iam::<account_id>:root is not authorized to perform: iam:DeactivateMFADevice on resource: * with an explicit deny in a service control policy

- What symptoms have you identified, if any: Root user cannot remove existing MFA devices

**Steps To Reproduce**
1. Sign in to the AWS console using a root user account
2. Select "Security Credentials" from the account menu in the top right hand corner
3. In the MFA section select the device you'd like to remove and click the "remove" button
4. You will be presented with a generic error message.  Running the browser in debug mode will reveal the response message indicating that iam:DeactivateMFADevice permissions are missing.

**Expected behavior**
The root user has the permissions required to add, remove and resync MFA devices.

**Additional context**
The issue appears to exist within the ASEA-Guardrails-Sensitive.json SCP:

```
{
      "Sid": "ROOT",
      "Effect": "Deny",
      "NotAction": [
        "iam:CreateVirtualMFADevice",
        "iam:EnableMFADevice",
        "iam:GetUser",
        "iam:ListMFADevices",
        "iam:ListVirtualMFADevices",
        "iam:ResyncMFADevice",
        "aws-portal:*",
        "sts:GetSessionToken",
        "iam:DeleteVirtualMFADevice",
        "trustedadvisor:*",
        "support:*"
      ],
      "Resource": "*",
      "Condition": {
        "ArnLike": {
          "aws:PrincipalARN": ["arn:aws:iam::*:root"]
        }
      }
}
```
The issue is fixed by adding "iam:DeactivateMFADevice":
```
{
      "Sid": "ROOT",
      "Effect": "Deny",
      "NotAction": [
        "iam:CreateVirtualMFADevice",
        "iam:EnableMFADevice",
        "iam:GetUser",
        "iam:ListMFADevices",
        "iam:ListVirtualMFADevices",
        "iam:ResyncMFADevice",
        "aws-portal:*",
        "sts:GetSessionToken",
        "iam:DeactivateMFADevice",
        "iam:DeleteVirtualMFADevice",
        "trustedadvisor:*",
        "support:*"
      ],
      "Resource": "*",
      "Condition": {
        "ArnLike": {
          "aws:PrincipalARN": ["arn:aws:iam::*:root"]
        }
      }
}
```
A similar issue was previously raised and fixed in Issue #686

**Comment by Brian969** [comment]
I believe this is by design and an intended security control. We do NOT want MFA removed from the root user, ever.  BTW: root user should virtually never be used, unless it is for one of the very few required activities, and then should only be allowed via MFA. Why do you want to remove MFA on the root user?

**Comment by evollebekk** [comment]
Our team is geographically dispersed and does not share the same secure physical location where a single key can be stored.  When several of our accounts were initially setup the usage of a virtual MFA token was used to provide a primary and secondary administrator with MFA access.  Now that IAM provides the ability to assign [multiple hardware MFA devices](https://aws.amazon.com/blogs/security/you-can-now-assign-multiple-mfa-devices-in-iam/) we've switched to hardware tokens to align with the ASEA best practices.  I would now like to remove the obsolete virtual MFA tokens that have been replaced by hardware devices.

The existing ASEA-Guardrails-Sensitive policy does already include iam:DeleteVirtualMFADevice permissions but unfortunately iam:DeactivateMFADevice is also required in order to remove a virtual MFA device.  I agree that removing an MFA token isn't something we'd typically like to occur and perhaps my issue in an edge case.  

One workaround to the issue could be to to temporarily define a custom override SCP with the missing permission so that the obsolete devices can be removed.
